tools/cloud/demo.sh

```diff
@@ -392,6 +406,27 @@ else
     else
         echo "Signals exist and are in use, continuing"
     fi
+
+    echo "Updating color attribute"
+    if UPDATE_SIGNAL_CATALOG_STATUS=`aws iotfleetwise update-signal-catalog \
+        ${ENDPOINT_URL_OPTION} --region ${REGION} \
+        --name ${SIGNAL_CATALOG_NAME} \
+        --description "DBC Attributes" \
+        --nodes-to-add '[{
+            "attribute": {
+                "dataType": "STRING",
+                "description": "Color",
+                "fullyQualifiedName": "Vehicle.OBD.Color",
```

Comment:
I don't think `Color` is part of OBD standard. Why not just name it `Vehicle.Color`?